jwst/associations/lib/dms_base.py

```diff
@@ -90,21 +90,13 @@
 
 IMAGE2_NONSCIENCE_EXP_TYPES = [
     'mir_coroncal',
-    'mir_tacq',
     'nis_focus',
-    'nis_tacq',
-    'nis_taconfirm',
-    'nrc_tacq',
-    'nrc_taconfirm',
     'nrc_focus',
-    'nrs_confirm',
     'nrs_focus',
     'nrs_image',
```

Comment:
EXP_TYPE's like "_focus", "_mimf", and "_image" should be processed as normal images, hence they should appear as science type members in a level-2b ASN for the purpose of ``calwebb_image2`` processing. The list here is labeled as "NONSCIENCE". Will they be treated as science types for the purpose of creating image2 type ASN's?

Reply:
All EXP_TYPE's found in `IMAGE2_NONSCIENCE_EXP_TYPES` do get Level2b processing through the rule `jwst.associations.lib.rules2b.Asn_Lv2ImageNonScience`. However, these exposures will not get things like backgrounds or imprint applied to them as "science" exposures would.  They will always be processed on their own.